src/vs/workbench/parts/debug/browser/debugCommands.ts

```diff
@@ -44,6 +44,29 @@ export function registerCommands(): void {
 		}
 	});
 
+	KeybindingsRegistry.registerCommandAndKeybindingRule({
```

Comment:
I would expect this command to be toggle enablement. So if my breakpoint is enabled to disable it and the other way around.
That way it is more flexible. In order to change that the name and the id should be changed.